microsoft/testsuites/core/storage.py

```diff
@@ -229,27 +240,40 @@ def verify_swap(self, node: RemoteNode) -> None:
         ),
     )
     def verify_resource_disk_io(self, node: RemoteNode) -> None:
-        resource_disk_mount_point = node.features[Disk].get_resource_disk_mount_point()
-
-        # verify that resource disk is mounted
-        # function returns successfully if disk matching mount point is present
-        node.features[Disk].get_partition_with_mount_point(resource_disk_mount_point)
+        node_disc = node.features[Disk]
+        resource_disks = node_disc.get_resource_disks()
+        if not resource_disks:
+            raise LisaException("Resource disk not found")
+        if "nvme" in resource_disks[0]:
+            raise SkippedException(
+                f"Resource disk type is NVMe and the VM has {len(resource_disks)} NVMe disks"  # noqa: E501
```

Comment:
The skip message doesn't explain why nvme disk should be skipped.

Reply:
Modified the exception messages for better understanding.